PRS3001-4000/PRS3137Cleopatr.xml

```diff
@@ -35,22 +35,34 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <langUsage><language ident="en">English</language></langUsage>
       </profileDesc>
       <revisionDesc>
-         <change who="PL" when="2016-04-28">batch updated according to requirements and issues.</change>
-         <change who="PL" when="2016-03-21"> Created file from
-                                                  google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: person</change>
+         <change who="PL" when="2016-03-21"> Created file from google spreadsheet </change>
+         <change who="PL" when="2016-04-28">batch updated according to requirements and issues.</change>
+         <change when="2023-05-03" who="CH">Added names.</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
             <person sex="2">
                <persName>
```

Comment:
wd:Q635 and add BM id to wikidata

Reply:
yes, all wikipedia entries - and many more - have wikidata IDs. just search wikidata. and add BM IDs.
https://betamasaheft.eu/Guidelines/?id=wikidata
(e.g. here example for Lebna Dengel 
![image](https://github.com/BetaMasaheft/Persons/assets/17987728/77bf4908-b270-4420-81cd-54a4982aafbd)
)
Ideally, all named entities that we have should also be on wikidata and linked in both directions.

Reply:
I think, I need to create an account and log-in. Confusingly, I have the opportunity to edit the page of Lebna Dengel on Wikidata.org, but I cannot change the page of Cleopatra before logging in.

Reply:
Unless I created an account in wikidata, I cannot edit the page of Cleopatra. For others, as for instance for Ptolemy Auletes, it works.

Reply:
I added the BM ID to Ptolemy Auletes (wd:Q39991=PRS14020Ptolemy) and to Hezekiah (wd:Q244912=PRS5330Hezekiah). But I was not able to edit the wikidata record for Cleopatra, that is not editable to me.

Reply:
Yes it is protected https://www.wikidata.org/wiki/Talk:Q635 only users with more than 50 edits may edit the page, I am not there either

Reply:
Do we know somebody with this requirement? Otherwise, we must skip it, don't we?

Reply:
I have left a request